libAvKys/Plugins/Crop/CMakeLists.txt

```diff
@@ -0,0 +1,66 @@
+# Webcamoid, webcam capture application. Crop Plug-in.
+# Copyright (C) 2022  Tj <hacker@iam.tj>
+#
+# Webcamoid is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Webcamoid is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Webcamoid. If not, see <http://www.gnu.org/licenses/>.
+#
+# Web-Site: http://webcamoid.github.io/
+
+cmake_minimum_required(VERSION 3.16)
+
+project(Crop LANGUAGES CXX)
+
+include(../../cmake/ProjectCommons.cmake)
+
+set(CMAKE_INCLUDE_CURRENT_DIR ON)
+set(CMAKE_AUTOUIC ON)
+set(CMAKE_AUTOMOC ON)
+set(CMAKE_AUTORCC ON)
+
+set(QT_COMPONENTS
+    Gui
+    Qml)
+find_package(QT NAMES Qt${QT_VERSION_MAJOR} COMPONENTS
+             ${QT_COMPONENTS}
+             REQUIRED)
+find_package(Qt${QT_VERSION_MAJOR} ${QT_MINIMUM_VERSION} COMPONENTS
+             ${QT_COMPONENTS}
+             REQUIRED)
+		 add_library(Crop SHARED
```

Comment:
I have seen this before in other commits, and I have passed them silently, but please, keep the code clean, don't add unnecessary spaces in the code, it make hard to read and understand the code.
There is no reason for adding spaces before ```add_library(```.

Reply:
That looks like some kind of spaces vs tabs glitch - its certainly not intentional. Not sure quite how those are getting in. I'll investigate QTCreator which I used for that, as opposed to `vim` for others.